test/functional/apps/visualize/_data_table.js

```diff
@@ -209,6 +209,29 @@ export default function ({ getService, getPageObjects }) {
       ]);
     });
 
+    it('should show correct data when selecting a field by its custom name', async () => {
+      await PageObjects.visualize.navigateToNewVisualization();
```

Comment:
Keeping my fingers crossed that this should help
```suggestion
      await PageObjects.visualize.navigateToNewAggBasedVisualization();
```